python_modules/dagit/dagit/cli.py

```diff
@@ -126,7 +126,9 @@ def _get_instance():
         with DagsterInstance.get() as instance:
             yield instance
     else:
-        with tempfile.TemporaryDirectory() as tempdir:
+        # make the temp dir in the cwd since default temp dir roots
+        # have issues with FS notif based event log watching
+        with tempfile.TemporaryDirectory(dir=os.getcwd()) as tempdir:
```

Comment:
does this have any cleanup issues if dagit crashes or something?

Reply:
same as before but more visible - if it segfaults it could but that should be unlikely given we are not executing user code in dagit process